flink-runtime/src/main/java/org/apache/flink/runtime/metrics/groups/AbstractMetricGroup.java

```diff
@@ -119,12 +122,28 @@ public AbstractMetricGroup(MetricRegistry registry, String[] scope, A parent) {
         this.variables = new Map[registry.getNumberReporters() + 1];
     }
 
+    CharacterFilter wrapWithDefaultFilter(CharacterFilter filter, char delimiter) {
```

Comment:
```suggestion
    private CharacterFilter wrapWithDefaultFilter(CharacterFilter filter, char delimiter) {
```

Reply:
oups I forgot to reduce visibilities, thanks for catching !